pipeline/pipeline.go

```diff
@@ -0,0 +1,33 @@
+package pipeline
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/m-lab/clinic2019/csvParser"
+
+	"github.com/m-lab/clinic2019/incident_viewer_demo/incident"
+)
+
+type IncidentProducer interface {
```

Comment:
what is the point of this interface?

Reply:
Removed since I didn't end up writing tests around it